core/core_integrations/slack.js

```diff
@@ -128,31 +128,30 @@ renameChannel = function(title, thread) {
 	}
 },
 
-getNextMessageId = teamId => {
-	return exports.config.slack_teams[teamId].event_id++;
-}
+getNextMessageId = function(teamId) {
+	return teamData[teamId].event_id++;
+},
 
 sendTyping = function(thread) {
-	var slackTeams = exports.config.slack_teams,
-		teamInfo = getChannelIdAndTeamId(thread),
+	var teamInfo = getChannelIdAndTeamId(thread),
 		teamId = teamInfo.team_id,
 		body = {
-			'type':'typing',
-			'channel': teamInfo.channel_id
+			type:'typing',
+			channel: teamInfo.channel_id
 		};
 
 	body.id = getNextMessageId(teamId);
 	body = JSON.stringify(body);
 
-	console.debug("slack-> send typing indication");
+	console.debug('slack-> send typing indication');
 	sendSocketMessage(teamId, body);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["sendSocketMessage" was used before it was defined](https://www.codacy.com/app/theprojectshop/Kassy/file/2549394133/issues/source?bid=3282987&fileBranchId=3288593#l147)